src/routes/safe/components/Settings/ManageOwners/AddOwnerModal/screens/Review/style.ts

```diff
@@ -64,7 +64,7 @@ export const styles = createStyles({
   selectedOwner: {
     padding: sm,
     alignItems: 'center',
-    backgroundColor: '#fff3e2',
+    backgroundColor: '#f7f5f5',
```

Comment:
This is not aligned with styles in Zepplin. It uses white background. But it's true that in replace owner it's using gray background for the new address.

I'm fine with both options, keeping consistency between Adding and Replace using gray background or white background

Reply:
I would stick with grey background. Especially in the replace owner modal I think it makes a clearer distinction between the remaining owner(s) and the added one.